presto-main/src/main/java/io/prestosql/execution/StageStateMachine.java

```diff
@@ -253,7 +259,7 @@ public BasicStageStats getBasicStageStats(Supplier<Iterable<TaskInfo>> taskInfos
         // information, the stage could finish, and the task states would
         // never be visible.
         StageState state = stageState.get();
-        boolean isScheduled = (state == RUNNING) || state.isDone();
+        boolean isScheduled = state.isRunning() || state.isDone();
```

Comment:
Maybe we should have `StageState#isScheduled` method instead.

Reply:
There is already a `scheduled` state for a stage - so having `isScheduled` could lead to ambiguity. Although, by that logic - the variable can be named differently (like `begunExecution`)